Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -998,6 +998,12 @@
     <!-- XHED: Title for the interop country list-->
     <string name="submission_interoperability_list_title">"The following countries currently participate in transnational exposure logging:"</string>
 
+    <!-- Submission Positive Other Warning No Consent -->
```

Comment:
There is German in here. Is that a problem?

Reply:
nope, no problem. Will be overwritten